app/src/main/java/de/westnordost/streetcomplete/quests/religion/AddReligionToPlaceOfWorshipForm.java

```diff
@@ -0,0 +1,99 @@
+package de.westnordost.streetcomplete.quests.religion;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.quests.ImageListQuestAnswerFragment;
+import de.westnordost.streetcomplete.view.Item;
+
+public class AddReligionToPlaceOfWorshipForm extends ImageListQuestAnswerFragment
+{
+	private static final int
+			MAX_DISPLAYED_ITEMS = 24, INITIALLY_DISPLAYED_ITEMS = 8;
```

Comment:
I think this is unnecessary. There are less than 24 items and getMaxNumberOfInitiallyShownItems could just return 8.

Reply:
Yes, it should be dropped in case of extracting only createItems rather than making an abstract class/

Reply:
done